src/engine/enginebuffer.h

```diff
@@ -8,6 +8,7 @@
 
 #include "audio/frame.h"
 #include "control/controlvalue.h"
+#include "engine/bufferscalers/enginebufferscalerubberband.h"
```

Comment:
do I get that right:
besides the other improvements in this PR, including only the scaler header (instead of the rubberband header) is more efficient because of
* pragma once (which we can't put in the rubberband header)?
* the scaler header is a little smaller than the rubberband header

Reply:
#pragma once is these days equal efficient than #ifndef RUBBERBAND_STRETCHER_H
The pure size might be an issue, but all this effect only the duration of the build run anyway. 

My driver for this change was hiding of information. In the current solution the scaler combobox work without any hard coded knowledge about the available scaler. 

There is however the issue with the "EngineBuffer::RUBBERBAND_FASTER" as default value. We may refactor this also into the engine buffer.

Than we are in the situation that we do not have to touch the preferences for any new scaler mode. 

What do you think? Shall I add this change as well?